doc/ref_impl/cntt-ri/chapters/chapter05.md

```diff
@@ -5,15 +5,248 @@
+Server template would be assigned to multiple servers, i.e physical hosts.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| template_name | String | Yes |  |
+| manufacturer | String | Yes |  |
+| model | String | Yes |  |
+| processor | String | Yes |  |
+| memory | String | Yes |  |
+| hard_drive | String | Yes |  |
+| raid | String | Yes |  |
+| network_controller_infos | List | Yes |  |
+| network_card_bond_infos | List | No | NIC bonding might not be always the case.|
+
+<p align="center"><b>Table 5-3-2:</b> Server template.</p>
+
+### 5.3.3 Server information
+Server will reference a server template, i.e, inherting all configuration of server template,
+Besides it may include additional information pim username, password, rack_name,position etc.
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| template_name | String | Yes | template referenced |
+| device_name | String | Yes | e.g NFV-D-HDBNJ-02A-3503-G-02-M-SRV-01 |
+| outband_ip | String | Yes |  |
+| outband_subnet_mask | String | Yes |  |
+| pim_username | String | Yes |  |
+| pim_password | String | Yes |  |
+| rack_name | String | Yes |  |
+| position | String | Yes |  |
+| remote_manange_ip | String | Yes | remote management ip, e.g. iLO,iDRAC, BMC |
+| remote_user | String | Yes | remote user  |
+| remote_password | String | Yes | remote password |
+
+<p align="center"><b>Table 5-3-3:</b> Server information.</p>
+
+
+### 5.3.4 Software configuration definition
+It includes Virtualized Infrastructure Manager configurations,
+
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| vim_name | String | Yes | vim name |
+| vim_id | String | No | a name VIM Resource pool naming rule |
+| vendor | String | Yes | VIM Provider information, e.g. ZTE,Huawei,Ericsson,NOKIA|
+| version | String | Yes | e.g NFV-D-HDBNJ-02A-3503-G-02-M-SRV-01 |
+| ip_version | String | Yes | Ipaddress type: IPV6 or IPV4 |
+| az_info | List | Yes | list of availability zone name designed for this VIM |
+| ha_name | List | No | HA name  if provided, HA is set of host server group having one or more same attribute(s),e.g: hardware specification.|
+| network_infos | List | Yes | List of NICs used in VIM deployment |
+| controller_nodes | List | Yes | List of controllers designed in VIM deployment |
```

Comment:
Is this a list of IPs ?

Reply:
a list of controllers' hostname.

Reply:
This section is for VIM configuration. controller hostnames won't be existing/known at the time of writing of this file since this file would be passed on to or would be used by some installer that would install VIM (may also provision the OS for controller nodes). 
Let me know if my understanding is correct ?

Reply:
Yes, partially agree. I thought this is totally at stage of designing &planning, and there should be a rule for node name planned, think about large scale case in production deployment. 
However, the requirement is that user should keep the relationship between VIM to be installed and nodes included.  So you mean we should keep ip list here instead? or Just remove it from VIM Configuration? 

If not proper in VIM configuration, we could create attribute of a reference of VIM on controller nodes.

Reply:
Do we need to add 'storage_nodes' attribute which would also be a List, similar to controller_nodes and compute_nodes? In the bottom of the document you have introduced storage_nodes anyaway